quattor/client/rpms.pan

```diff
@@ -29,6 +28,11 @@ include { 'rpms/package_default_versions' };
     if(! exists(spma)) {
         SELF[spma] = nlist();
     };
+    ncd = escape('ncm-ncd');
```

Comment:
That was how pkg_add() used to work, in my initial note I wrote pkg_repl() instead which used to do the right thing. However upon reviewing the most recent function definitions I came to the conclusion that pkg_add() should also now work, but maybe I am wrong. 

Either way, we should have a function that does this rather than hacking it into each piece of code like this. :-)